app/konnect/gateway-manager/index.md

```diff
@@ -56,9 +56,6 @@ entity-specific permissions. For more information, see [Administer teams](/konne
 
 ### {{site.base_gateway}} control planes
 
-Every region in every organization starts with one default control plane.
```

Comment:
Why delete the first line? It's still accurate.

Reply:
I think I saw that we're going to stop provisioning the default CP too

Reply:
Looks like that hasn't gone live yet: 

> we will also stop provisioning the default control plane in each region soon 

https://kongstrong.slack.com/archives/C04RXLGNB6K/p1723651386893509?thread_ts=1723650140.813759&cid=C04RXLGNB6K